### ISSUE TITLE
Publication info popup dialog (metadata), punctuation and separators should be localized

### ISSUE THREAD

**Issue by danielweck** [edrlab/thorium-reader#962]
For example, in French, colon separator is between space characters (before and after), whereas currently there is a space only after (which is fine for English locale).

![Thorium_publication-metadata-id_1](https://user-images.githubusercontent.com/625201/75250221-c5681180-57cf-11ea-958d-3c7055110f9b.png)

![Thorium_publication-metadata-id_2](https://user-images.githubusercontent.com/625201/75250225-c731d500-57cf-11ea-9327-edb1e8548925.png)

**Comment by panaC** [comment]
identifier has been hided

**Comment by danielweck** [comment]
Re-opening the issue (this isn't about hiding the UUID).

See screenshots in English and French that pin-point where the missing spaces are (English OK, French
lacks separator spaces):

<img width="912" alt="Screenshot 2020-10-14 at 15 23 39" src="https://user-images.githubusercontent.com/625201/96002987-ab983f00-0e31-11eb-9681-3c3170a667b2.png">

<img width="912" alt="Screenshot 2020-10-14 at 15 25 26" src="https://user-images.githubusercontent.com/625201/96002976-a804b800-0e31-11eb-9f96-4b8be0d0a65b.png">

**Comment by panaC** [comment]
very annoying this issue because it's need to modify all locale file. Colon seperator is hardcoded !

**Comment by danielweck** [comment]
Yes, the translated labels should all have interpolations for real localisation / globalisation, accounting not only for token positioning but also date format, etc.

**Comment by PhilippeBruno** [comment]
I can't believe this has not been fixed after all these years. Anyway, another problem is the format of the publication date. Currently, it seems to be hardcoded to MM/DD/YYYY which is not suitable for everyone. Please fix this because it is confusing as shown in the attached example.

<img width="1165" height="957" alt="Image" src="https://github.com/user-attachments/assets/729f762e-24d1-45ba-8e8a-d96569d2b869" />
